src/activities/content/ContentHtmlFileTemplatesEntity.js

```diff
@@ -1,14 +1,21 @@
-import { Entity } from '../../es6/Entity';
-
-/**
- * ContentHtmlFileTemplatesEntity class representation of an html file templates entity.
- */
-export class ContentHtmlFileTemplatesEntity extends Entity {
-
-	/**
-	 * @returns {array|undefined} Array containing the subentites of the file template entity, if they exist
-	 */
-	getHtmlFileTemplates() {
-		return this._entity && this._entity.getSubEntitiesByRel('about');
-	}
-}
+import { Entity } from '../../es6/Entity';
+
+/**
+ * ContentHtmlFileTemplatesEntity class representation of an html file templates entity.
+ */
+export class ContentHtmlFileTemplatesEntity extends Entity {
+
+	/**
+	 * @returns {string|undefined} Title of the content HTML file template entity
+	 */
+	title() {
```

Comment:
I'm not sure why there are all these whitespace changes, this should be the only changes (I have LF line endings)

Reply:
i did a look at some of our files in this repo and it's a mix of CRLF and LF, I'm guessing this repo doesn't have a linter rule for it, so it's probably just using who ever made the file and if someone with a different setting edits it, if their editor has it set to, it will overwrite that file. but yeah, seems like there isn't a standard set, even looking at some files that don't belong to us